### ISSUE TITLE
extension built by meson are not packaged in wheel

### ISSUE THREAD

**Issue by kif** [FFY00/meson-python#80]
Hi,
I managed to mimic the packaging of scipy on another project and despite the binary extensions are built, they are not packaged in the wheel and provide this warning message:
```
WARNING Using heuristics to map files to wheel, this may result in incorrect locations
WARNING File could not be mapped to an equivalent wheel directory: /tmp/build-via-sdist-jsg87qg9/fabio-0.15.0-a0/.mesonpy-5bwmoite/install/usr/lib/python3/dist-packages/fabio/ext/cf_io.cpython-39-x86_64-linux-gnu.so ({moduledir_shared}/cf_io.cpython-39-x86_64-linux-gnu.so)
```
Did I forgot any configuration ? Since the same "bug" occurs when packaging scipy, I suspect there is something in meson-python (which I don't master for the moment).

Cheers,

Jerome

**Comment by rgommers** [comment]
This should be fixed by gh-77 shortly.

**Comment by kif** [comment]
Indeed, I confirm this patch fixes the bug. 

One needs to build the package with "python -m build -n" to use the currently version of meson-python and not the one packaged which does not yet contain the fix.
The bug can be closed when this patch is merged.

**Comment by FFY00** [comment]
Fixed by #77. 0.6.0 should be released soon with this fix.
